Dockerfile

```diff
@@ -2,13 +2,10 @@ FROM ubuntu:14.04
 
 # Install deps
 RUN apt-get -q update \
-		&& apt-get install -y git lib32stdc++6 python build-essential lib32z1 wget ca-certificates --no-install-recommends
+		&& apt-get install -y git lib32stdc++6 python python-pip build-essential lib32z1 wget ca-certificates --no-install-recommends
 
-# Install S3 tool
-RUN wget -O- -q http://s3tools.org/repo/deb-all/stable/s3tools.key | sudo apt-key add - \
-		&& wget -O/etc/apt/sources.list.d/s3tools.list http://s3tools.org/repo/deb-all/stable/s3tools.list \
-		&& apt-get -q update \
-		&& apt-get install -y s3cmd \
+# Install AWS CLI
+RUN pip install awscli \
 		&& apt-get clean \

```

Comment:
I just tried to find, which dir is used as pip cache, but didn't see anything in the Docker container. Normally I would expect "~/.cache/pip", but this doesn't exists at all. Maybe you know the right dir?